setup.py

```diff
@@ -43,7 +43,7 @@
             script = 'gips_%s = gips.scripts.%s:main' % (name, name.lower())
             console_scripts.append(script)
     except:
-        print traceback.format_exc()
+        print(traceback.format_exc())
 
 
```

Comment:
See if there is anything that we need to do to specify that we are python 3 or greater only.